.bazelrc

```diff
@@ -80,6 +80,14 @@ build:sanitizer-common --copt="-fno-omit-frame-pointer" --copt="-mno-omit-leaf-f
 build:asan --config=sanitizer-common
 build:asan --copt="-fsanitize=address" --linkopt="-fsanitize=address"
 build:asan --test_env=ASAN_OPTIONS=abort_on_error=true
+# Exclude benchmarks from asan build due to large size (exceed github runner limits)
+build:asan -- -//src/workerd/tests:bench-api-headers
```

Comment:
Probably want to create the following
```bazelrc
build:limit-storage -- -//src/workerd/tests:bench-api-headers
build:limit-storage -//src/workerd/tests:bench-global-scope
build:limit-storage -//src/workerd/tests:bench-json
build:limit-storage -//src/workerd/tests:bench-kj-headers
build:limit-storage -//src/workerd/tests:bench-mimetype
build:limit-storage -//src/workerd/tests:bench-regex
build:limit-storage -//src/workerd/tests:bench-tools

build:asan --config=limit-storage
build:debug --config=limit-storage
```

Reply:
> Probably want to create the following
> 
> ```
> build:limit-storage -- -//src/workerd/tests:bench-api-headers
...
> ```

Thanks Sam, moved this config into `test.yml` so it only affects builds on github ci.